lib/compiler-llvm/src/translator/code.rs

```diff
@@ -9344,6 +9344,213 @@ impl<'ctx, 'a> LLVMFunctionCodeGenerator<'ctx, 'a> {
                 size.add_attribute(AttributeLoc::Function, self.intrinsics.readonly);
                 self.state.push1(size.try_as_basic_value().left().unwrap());
             }
+            /***************************
+             * Reference types.
+             * https://github.com/WebAssembly/reference-types/blob/master/proposals/reference-types/Overview.md
+             ***************************/
+            Operator::RefNull { .. } => {
+                self.state.push1(self.intrinsics.funcref_ty.const_null());
+            }
```

Comment:
This function takes a `type` which will tell you if it's a funcref or an externref

Reply:
Good catch, thanks. Fixed!